pkg/actor/partitioned_update.go

```diff
@@ -260,6 +260,10 @@ func getStsAnnotation(statefulSet *appsv1.StatefulSet, key string) string {
 }
 
 func getImageNameNoVersion(image string) string {
+	// if somehow this arrives as sha256 we do not extract version
```

Comment:
@udnay  I added some unit tests, but there should not be any negative cases here, because the image it is validated by the version checker job first and it is added by our workflow.  It was a bug in our code because I did not treated the case of Openshift where they replace the supported images with the sha256.

Reply:
Then why not put that code into the version checker if it can already do parsing and validating of the versions?

Reply:
The validation consist in running a job with the image the user sets. First step is checking that the job is running. If the job is running the image exists. Next we check to see if the image is a cockroachdb image by running an exec on the pod and trying to obtain the version tag. If also  this step succeeds we consider it a valid image.